### DISCUSSION TITLE
How can I talk to someone from legal departement at Github?

### DISCUSSION THREAD

**Discussion by LawlessLawless** [community/community#34756] [Other Feature Feedback, Questions, & Ideas]
I did not found any email adress dedicated.

**Comment by skadomsky** [comment]
If there is a pubilc repository available in their profile, try [using issue feature](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) to mention them.
Who else would use GitHub for communicating?

**Comment by jfullstackdev** [comment]
it's maybe implied here 

![image](https://user-images.githubusercontent.com/47092464/193404698-60cf9e04-3b94-4fe5-b3d6-41da066e2515.png)

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#disclaimer

as you can see, there is that disclaimer and there is no warranty, you may want to get your own legal professional for that
